### DISCUSSION TITLE
The license does not match the Video DownloadHelper browser version

### DISCUSSION THREAD

**Discussion by minterty** [aclap-dev/video-downloadhelper#1519] [License & Purchase requests]
### What's your issue?

Hello,

I have just bought license using Microsoft Edge and paid. But when I clicked on "Install" I have got the message: "The license does not match the Video DownloadHelper browser version". I have tried both Chrome and Edge.

Please advise.

Thank you!

### The 10 first digits (only 10) of your license number or email address

cb4b063e69

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You may have installed the wrong extension. 
I recommend that you remove the extension and reinstall it from here: https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn
